packages/cli/src/auth/methods/ldap.ts

```diff
@@ -54,7 +56,10 @@ export const handleLdapLogin = async (
 				user_type: 'ldap',
 				was_disabled_ldap_user: false,
 			});
-			return user;
+			const authUser = await Container.get(AuthUserRepository).findForAuth({
+				id: user.id,
+			});
+			return authUser!;
```

Comment:
Maybe we could have a separate method on `AuthUserRepository` for "promoting"/"escalating" a user to auth user that fetches it from DB by ID but throws if user is not found. That way there's a clear method for doing that and there's no need for the type assertion